src/client/components/CmpConsentStateHiddenInput.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+type CmpConsentedStateProps = {
+  cmpConsentedState: boolean;
+};
+
+export const CmpConsentedStateHiddenInput = ({
+  cmpConsentedState,
+}: CmpConsentedStateProps) => {
+  return (
+    <>
+      <input
+        type="hidden"
+        name="_cmpConsentedState"
+        value={String(cmpConsentedState)}
```

Comment:
This can be a template string instead.

```suggestion
        value={`${cmpConsentedState}`}
```